woocommerce/woocommerce
# Check order type is set before returning to prevent notice.

### All Submissions:






### Changes proposed in this Pull Request:



When getting order type of a non-order ID, we were emitting a warning. This PR fixes the warning by adding a check to make sure that order type exists before returning.






### How to test the changes in this Pull Request:



1. Create a product, note its ID.
2. Run the code ` Automattic\WooCommerce\Utilities\OrderUtil::is_order( $product_id );`
3. On the trunk, you will see a warning. On this PR, no warning will be emitted.



### Other information:




### FOR PR REVIEWER ONLY:
